instrumentation/aws-lambda-1.0/library/aws-lambda-1.0-library.gradle

```diff
@@ -17,8 +17,12 @@ apply from: "$rootDir/gradle/instrumentation-library.gradle"
 
 dependencies {
   library group: 'com.amazonaws', name: 'aws-lambda-java-core', version: '1.0.0'
+  library group: 'com.amazonaws.serverless', name: 'aws-serverless-java-container-core', version: '1.5.2'
+
   compileOnly deps.opentelemetrySdk
 
+  compile('com.fasterxml.jackson.core:jackson-databind:2.10.3', 'com.fasterxml.jackson.module:jackson-module-afterburner:2.9.10')
```

Comment:
I don't think we need afterburner since we're not mapping to POJO, just reading the tree.

Reply:
There is a mapping call fetching Headers class instance - `return OBJECT_MAPPER.readValue(jParser, Headers.class);` I believe that's optimised with afterburner (although I'm not expert here ;) ).

Reply:
Ah I missed that. That being said, since we're telemetry I don't think we should try for "more than the base". Do you know if aws SDK uses afterburner? If they do no problem adding it :) But if not we should try to stick to their dependence as much as possible and I would avoid it.

Reply:
Noted for future assignments, thanks :) Here it's ok as the afterburner is references by the SDK (in particular by 'aws-serverless-java-container-core').